templates/inputs.cue

```diff
@@ -77,8 +77,18 @@ _#ControllerConfig: {
 	enabled: *true | bool
 	_#Common
 	ingressClassName: *null | string
+	hosts:            #HostsConfig
+	secretName:       *null | string
 }
 
+#HostsConfig: [{
```

Comment:
Is there any reason not to use the original k8s API here? https://pkg.go.dev/k8s.io/api/networking/v1#HTTPIngressPath